CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # bedrock-profile ChangeLog
 
+## 13.0.0 - 2021-TBD
```

Comment:
I didn't know `lib/kms.js` was supposed to be an internal file. It was being required in `test/mocha/helpers.js` in `bedrock-vc-issuer` https://github.com/digitalbazaar/bedrock-vc-issuer/blob/aef21b1a066e16d9052e2ed2af4ad6467c72f192/test/mocha/helpers.js#L19. So, I thought it might be used elsewhere too and if not updated would not work with the latest `kmsClient.getKeystore` changes. If this is not being used here or anywhere else other than the `test/mocha/helpers.js` file in bedrock-vc-issuer, should I remove it?

Reply:
Honestly, I'm not sure it's even worth keeping this function around (it saves like one extra line of setup, and is only used in that test file.)